osu.Game/Screens/Play/InGameScoreContainer.cs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using Humanizer;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Scoring;
+using osu.Game.Users;
+using osuTK;
+
+namespace osu.Game.Screens.Play
+{
+    public class InGameScoreContainer : FillFlowContainer<InGameScoreItem>
```

Comment:
`ScoreContainer`

Reply:
same as above, there is `PaginatedScoreContainer` for example